### PR TITLE
docs: add operationId in spec

### PR DESCRIPTION
**Pull request by jannyHou**
connect to https://github.com/strongloop/loopback-next/issues/5970
Add guide for specifying `operationId` in the spec

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Controllers.md [hunk 1]
```diff
@@ -196,6 +196,49 @@ export class HelloController {
...
+  name = 'operationIdEnhancer';
+  // takes in the current spec, modifies it, and returns a new one
+  modifySpec(spec: OpenApiSpec): OpenApiSpec {
+    const paths = spec.paths;
+    for (const path in paths) {
+      for (const op in path) {
+        const operationId = paths[path][op].operationId;
+        // change operationId from 'MyController.MyMethod' to
+        // 'MyController-MyMethod'
+        if (operationId) operationId.replace('.', '-');
```
Comment (raymondfeng) [inline docs/site/Controllers.md:234]: The updated value needs to be assigned to `paths[path][op].operationId`

Comment (jannyHou) [reply docs/site/Controllers.md:234]: good catch.

**Comment by raymondfeng** [comment]
Code lint fails - https://travis-ci.com/github/strongloop/loopback-next/jobs/371886121

**Comment by jannyHou** [comment]
Fixed the lint problem.

**Pull request merged**